lib/chef/resource/user.rb

```diff
@@ -42,7 +42,7 @@ def initialize(name, run_context = nil)
         @force = false
         @non_unique = false
         @supports = {
-          :manage_home => false,
+          :manage_home => true,
```

Comment:
Yeah, if I understand #604 correctly the problem is that we've always behaved like manage_home was forced to true and the button has never worked.  This fixes it, but we need to set manage_home to false to not have an API break.